internal/wasm/gofunc_test.go

```diff
@@ -10,6 +10,9 @@ import (
 	"github.com/tetratelabs/wazero/internal/testing/require"
 )
 
+// testCtx is an arbitrary, non-default context. Non-nil also prevents linter errors.
+var testCtx = context.WithValue(context.Background(), struct{}{}, "arbitrary")
```

Comment:
ps if wondering I added this once per package instead of exporting a test utility. If we feel strongly about it we can do that in the testing package.